bioconda_utils/pkg_test.py

```diff
@@ -0,0 +1,104 @@
+import subprocess as sp
+import tempfile
+import tarfile
+import os
+import shlex
+
+from conda_build.metadata import MetaData
+
+'mulled-build build-and-test "gffutils" --test "python -c \'import gffutils\''
+
+
+
+
+
+def get_tests(path):
+    "Extract tests from a built package"
+    tmp = tempfile.mkdtemp()
+    t = tarfile.open(path)
+    t.extractall(tmp)
+    input_dir = os.path.join(tmp, 'info', 'recipe')
+
+    tests = []
+    recipe_meta = MetaData(input_dir)
+
```

Comment:
It's just that to use `galaxy.tools.deps.mulled.mulled_build.get_tests` I'd need to provide an `args` object with `.recipes` attribute, that's all. 

Also since the package testing in this module is intended to work on the level of individual tarballs, I figured I'd give it the tarball directly rather than an unpacked recipe dir. Those two things made it easier to just write a new function.

Is there another `get_tests` I'm missing from elsewhere though?